data/json/items/containers.json

```diff
@@ -271,7 +271,7 @@
         "description": "A standard plastic jug used for milk and household cleaning chemicals.",
         "price": 0,
         "weight": 190,
-        "volume": 15,
+        "volume": 18,
```

Comment:
I want to overhaul all the containers and volumes eventually but I want to do this in stages. The gallon jug is changed because its the container used by wine (a precursor to ethanol).